AirCasting/SessionViews/MultiSliderView.swift

```diff
@@ -6,19 +6,24 @@
 //
 
 import SwiftUI
+import Resolver
 
 struct ThresholdsSliderView: View {
     
     @ObservedObject var threshold: SensorThreshold
+    var selectedStream: MeasurementStreamEntity?
```

Comment:
we're trying to move away from using `CoreData` objects directly in `View`s. Lets come up with something properly separated (applies to all `View`s in this PR that introduce `MeasurementStreamEntity`)